h2o-genmodel/src/main/java/hex/genmodel/easy/error/VoidErrorConsumer.java

```diff
@@ -0,0 +1,15 @@
+package hex.genmodel.easy.error;
+
+import hex.genmodel.easy.EasyPredictModelWrapper;
+
+/**
+ * A void implementation of {@link hex.genmodel.easy.EasyPredictModelWrapper.ErrorConsumer}.
+ * It's purpose is to avoid forcing developers do to null checks in code before each and every call.
+ */
+public class VoidErrorConsumer implements EasyPredictModelWrapper.ErrorConsumer {
+
+  @Override
+  public void dataTransformError(String columnName, Object value, String message) {
```

Comment:
Hope is this method will be

- Inlined by JIT, thus avoiding new stack frame
- Calling the method will be completely avoided when JIT kicks in and [dynamic deoptimization](http://www.oracle.com/technetwork/java/whitepaper-135217.html) used in Debug mode.

Reply:
I think inlining will happen only if JVM has loaded just one instance of the class that can potentially be called (one implementation of the interface).

I am not sure if this setup inlining will actually happen if the user has loaded two instances of the interface.

Marking the method final here wouldn't hurt.

It is not a big concern creating the hashmap will be much more expensive than calling the method.